typography/typography.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="author" content="Parth Pandya" />
+    <meta name="description" content="Typography Docs" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <title>Typography</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <h1 class="head-start">FoundOve UI</h1>
+      </nav>
+
+      <div class="pg-section">
+        <h2>Typography</h2>
+        <hr />
+
+        <ul>
+          <li class="sub-title">Headings</li>
+          <div class="disp-typo">
+            <h1 class="h-1">Heading 1</h1>
+            <h2 class="h-2">Heading 2</h2>
+            <h3 class="h-3">Heading 3</h3>
+            <h4 class="h-4">Heading 4</h4>
+            <h5 class="h-5">Heading 5</h5>
+          </div>
+          <li class="sub-title">Texts</li>
+          <div class="disp-typo">
+            <p class="h-1">Lorem Ipsum</p>
+            <p class="h-2">Lorem Ipsum</p>
+            <p class="h-3">Lorem Ipsum</p>
+            <p class="h-4">Lorem Ipsum</p>
+            <p class="h-5">Lorem Ipsum</p>
+          </div>
+          <li class="sub-title">Texts</li>
```

Comment:
"Text weights" would be a better title for this as the previous section also has the title "Texts".
Also, do mention the weight numbers on the side or below each of them.

Reply:
Cool, It'll be also done. Thanks for being up here :+1: